### ISSUE TITLE
minZoom and maxZoom only seem to work in options

### ISSUE THREAD

**Issue by dantistus** [sandydoo/ember-google-maps#46]
### Complete this checklist before opening an issue

*Please read the documentation before opening an issue*. Once you understand how this addon works, you will find that most problems can be solved by consulting the [Google Maps API reference](https://developers.google.com/maps/documentation/javascript/reference/3.exp/).

* [x] I've read the relevant documentation for this addon on [ember-google-maps.sandydoo.me](https://ember-google-maps.sandydoo.me/docs/getting-started).

* [x] I understand how to set options on the components in this addon. Options are not hardcoded. Arguments passed to a component are used to set the options on the map component.

* [x] I've searched the relevant section of the [Google Maps API reference](https://developers.google.com/maps/documentation/javascript/reference/3.exp/).

* [x] I've [searched the closed issues](https://github.com/sandydoo/ember-google-maps/issues?q=is%3Aissue+is%3Aclosed) for similar problems.

--------------------------------------------------------------------------------

### Describe the bug
This page https://ember-google-maps.sandydoo.me/docs/map reads: 

> The g-map component accepts all of the GoogleReference::MapOptions options you would pass to a Google Map instance. These are automatically watched for changes.

And there is an example with `minZoom`. But this doesn't seem to work, the map pugin seems to respect `minZoom` and `maxZoom` only when passed in options object.

### To Reproduce
Does not work:
```
{{#g-map
  minZoom=minZoom
  maxZoom=maxZoom
}}
```

Works:
```
options = {
  minZoom: minZoom,
  maxZoom: maxZoom
}

{{#g-map
  options=options
}}
```

--------------------------------------------------------------------------------

### Addon configuration and device details

**Addon configuration**:

 - Addon version: 2.0.5

**Comment by sandydoo** [comment]
@dantistus, I can't seem to replicate this. I've tested both `minZoom` and `maxZoom`, setting the value directly, via a variable and via a computed property.
I don't really have any ideas on what could be wrong. Is that `minZoom` variable you're using in scope in the template? Does the browser console print the value when you add `{{log minZoom}}` in your template?

The example with `minZoom` in the docs is misleading, since I never actually set those options on the map you see. I've updated the map to reflect the docs, thanks for letting me know.
